spec/plsql/package_spec.rb

```diff
@@ -20,7 +20,7 @@
         END test_procedure;
       END;
     SQL
-
+    plsql.execute("grant execute on TEST_PACKAGE to #{DATABASE_USERS_AND_PASSWORDS[1][0]}")
```

Comment:
> That got me thinking about an edge case - what if we connect through proxy user? (I'm not sure whether the other tests would be compatible with such scenario either).

Looks like the other tests already expect the username used to connect to match session user: 
[schema_spec.rb#L37](https://github.com/rsim/ruby-plsql/blob/e406170cf98f3e37721b828bd83179db01e604d2/spec/plsql/schema_spec.rb#L37).

So I think it's OK to rely on this assumption for now. We may think of how to address proxy user connections some other time in future.